packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -4114,8 +4119,8 @@ void main() {
       selection,
       equals(
         const TextSelection(
-          baseOffset: 20,
-          extentOffset: 55,
+          baseOffset: 0,
+          extentOffset: 72,
```

Comment:
This test appeared to be incorrect based on my observations on Mac.  It moves the uppermost offset, not necessarily the extent.

On Windows and Ubuntu, I observed that this key combination did nothing, but I left Flutter's behavior alone, which is to do the same thing as Mac.  I'll keep this in mind as we continue to work on desktop text editing and will consider getting rid of these shortcuts.